waypoints/src/main/resources/lang/zh-hk.yml

```diff
@@ -0,0 +1,714 @@
+      activeColor: "<gold>"
+      inactiveColor: "<gray>"
+
+    settings:
+      displayName: "<yellow>設定"
+      description: |
+        <gray>更改所有座標的可見性和指示器的使用
+
+    deselect:
+      displayName: "<yellow>取消選擇所有座標"
+      description: "<gray>導航消息將不再提醒"
+      selected: "<gray>當前選擇的座標:"
+      names:
+        waypoint:
+          private: "<dark_purple><path>"
+          death: "<red>死亡 <created_at:'dd.MM.uuuu kk:mm'>"
+          public: "<aqua>公開/<path>"
+          permission: "<light_purple>權限/<path>"
+          temporary: "<dark_purple>臨時座標"
+        playerTracking: "<blue>玩家: <name>"
+        unknown: "<red>未知"
+
+    setWaypoint:
+      displayName: "<yellow>創建一個新的座標"
+      description: |
+        <gray>座標將會創建在當前的座標組內
+        
+        <gray>若按下 <yellow>shift+左鍵<gray>, 你可使用坐標創建座標, 例如:
+        <green>102 35 -238
+    createFolder:
+      displayName: "<yellow>創建新座標組"
+      description: | 
+        <red>請注意，這不是創建座標（創建座標請選擇火把）。
+        <red>這只是創建座標的文件夾，方便你為座標歸類。
+
+  settings:
+    toggleGlobals:
+      visible:
+        displayName: "<yellow>所有座標可見"
+        description: "<gray>點擊隱藏"
+      hidden:
+        displayName: "<yellow>所有座標隱藏"
+        description: "<gray>點擊顯示"
+    temporaryWaypoints:
+      receivable:
+        displayName: "<yellow>臨時座標可接收"
+        description: |
+          <gray>其他玩家可用:
+          <green>/waypoints setTemporary X Y Z <name>
+          <gray>指令發送臨時座標給你
+          
+          <gray>點擊此處以阻止它們
+      blocked:
+        displayName: "<yellow>臨時座標已關閉"
+        description: |
+          <gray>其他玩家可用:
+          <green>/waypoints setTemporary X Y Z <name>
+          <gray>指令發送臨時座標給你
+          
+          <gray>點擊此處以再次接收它們
+    pointers:
+      title:
+        displayName: "<yellow>自定義指示器的使用"
+        description: ""
+      on:
+        displayName: "<bold><green><name>"
+        description: ""
+      off:
+        displayName: "<bold><red><name>"
+        description: ""
+      actionBar:
+        name: "動作條"
+        description: |
+          <gray>在你的動作條上顯示一個指示器,
+          <gray>告訴你應朝哪個方向看以到達座標
+      beacon:
+        name: "烽火台"
+        description: |
+          <gray>當你進入座標的視距後,
+          <gray>烽火台光柱會在座標上方顯示, 可自定義顏色
+      blinkingBlock:
+        name: "閃爍方塊"
+        description: |
+          <gray>當你進入座標附近時,
+          <gray>閃爍方塊會在座標上方顯示確切位置
+      bossBar:
+        name: "Boss狀態血條"
+        description: |
+          <gray>在Boss狀態血條中顯示一個指南針,
+          <gray>並標記你選擇的座標
+      compass:
+        name: "指南針"
+        description: |
+          <gray>你物品欄中的任何指南針現在都指向該座標
+      hologram:
+        name: "全息圖"
+        description: |
+          <gray>一個全息圖懸浮在你附近, 指向座標的方向,
+          <gray>顯示名稱、圖標和距離
+      particle:
+        name: "粒子指南針"
+        description: |
+          <gray>你腳下的粒子顯示座標所在的方向
+      trail:
+        name: "粒子軌跡"
+        description: |
+          <gray>計算並顯示通往座標的
+          <gray>可能路徑, 用粒子軌跡顯示
+  # amount: Amount of waypoints in this global folder visible for the player

```

Comment:
Empty newline above went missing (I like to keep the spacing between entries across languages identical)